src/components/Form.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <div>
+      <input type="text" placeholder="book name" />
```

Comment:
[OPTIONAL]

- [ ] Defining input fields inside a form tag is one of the best practices for writing sematic tags

Reply:
thank you!